### ISSUE TITLE
Names of the datasets

### ISSUE THREAD

**Issue by Aktsvigun** [codalab/codabench#1238]
Hi team,

Can I please clarify the dataset names you use in your API. Do I understand it properly that:

- `Public data`: this is the train data that can be used to train a model
- `Input data`: the inputs (texts / images / etc.) that are used in the current phase, which need to be predicted
- `Reference data`: the labels corresponding to the input data

If the above "dictionary" is true, I also wonder, how can I set to share both `Public data` and `Input data` with the users in `competition.yaml`? Thanks!

**Comment by ihsaan-ullah** [comment]
`Public data`: training data that can be released publicly so that participants can use it in whatever way they want

`Input data`: combination of training data (data and labels) and test data (only data, no labels)

`reference data`: test labels

People can design competitions that accept code submissions or result submissions or both. Usually for competitions with just `result submission` they distribute the whole input data and the participants submit test set predictions as result submission. Note that in this case you can call input data as public data.

In codabench, by design input data is for internal use so you can only share public data or you can just make a combined zip of train + test data and distribute it as public data

**Comment by Didayolo** [comment]
@Aktsvigun @ihsaan-ullah 

> In codabench, by design input data is for internal use so you can only share public data or you can just make a combined zip of train + test data and distribute it as public data

I believe that:

- By default public data is available to participant
- By default reference data is NOT available to participant (being the ground truth)
- Input data can be made available using the `make_data_available: True` setting

**Comment by Didayolo** [comment]
Not sure if this "Files available" feature is documented in the Wiki
